arm/Microsoft.ServiceFabric/clusters/.bicep/nested_rbac.bicep

```diff
@@ -0,0 +1,11 @@
+param roleAssignmentObj object
```

Comment:
Please update his structure to the latest in upstream main

Reply:
Updated.